api/pkg/apps/webapp/teams.go

```diff
@@ -65,6 +65,20 @@ func (s *Server) Team(w http.ResponseWriter, req *http.Request) {
 		return
 	}
 
+	n, err := iamClient.Nationalities().List(ctx, iam.NationalityListOptions{
+		TeamID: id,
+	})
+	if err != nil {
+		s.Error(w, err)
+		return
+	}
+
+	c, err := iamClient.Countries().Get(ctx, n.Items[0].CountryID)
```

Comment:
Can we add a check here on the n.Items to make sure that its len is bigger than 0 before we address n.Items[0], this may lead to a panic like we saw earlier